package.json

```diff
@@ -240,7 +240,7 @@
   },
   "dependencies": {
     "@amilajack/react-plyr": "^2.1.2",
-    "@amilajack/yifysubtitles": "^2.1.2",
+    "@amilajack/yifysubtitles": "github:amilajack/yifysubtitles",
```

Comment:
Why not publish it in npm ?

Reply:
Yea I should probably use NPM instead. The reasoning for this was that this module is pretty experimental so I didn't want to manage bumping the semver every time I fix something. Good call